.travis.yml

```diff
@@ -2,18 +2,20 @@ group: travis_latest
 language: python
 matrix:
     include:
-      # We run Python 3.8 based tests towards the start. This is where additional tests are run so we give it more time
-      - python: 3.8
+      # We run Python 3.9 based tests towards the start. This is where additional tests are run so we give it more time
+      - python: 3.9
         dist: xenial
         sudo: true
```

Comment:
do you know what this sudo thing is for?

Reply:
It is required when specifying a particular version of Ubuntu. But we don't need to do that anymore. Here's a PR to remove it - #1646.